src/styles/main.scss

```diff
@@ -3,6 +3,7 @@
 @import "../javascripts/components/navbar.scss";
 @import "../javascripts/components/userList.scss";
 @import "../javascripts/components/addMessage.scss";
+@import "../javascripts/components/largeText.scss";
```

Comment:
This line is fine, I just had to put a comment somewhere in this file...

There's something I added in my last branch that's screwing up the checkbox showing up right. If you remove `position: fixed` from line 240 under `#compose-area` that should let it show up correctly.